draft-lenders-dns-cbor.md

```diff
@@ -851,9 +869,9 @@ Reference: \[TBD-this-spec\]
 In the registry "{{cbor-tags (CBOR Tags)<IANA.cbor-tags}}" {{IANA.cbor-tags}},
 IANA is requested to allocate the tags defined in {{tab-tag-values}}.
 
-|    Tag | Data Item        | Semantics                     | Reference              |
-|   TBDt | unsigned integer | DNS name suffix extension     | draft-lenders-dns-cbor |
-| TBD141 | array            | CBOR EDNS option record       | draft-lenders-dns-cbor |
+|    Tag   | Data Item | Semantics                                                            | Reference              |
+| TBD141   | array     | CBOR EDNS option record                                              | draft-lenders-dns-cbor |
+| TBD28259 | any       | Packed CBOR; implicit text string suffix sequence shared-item table  | draft-lenders-dns-cbor |
```

Comment:
everything except `tag`, `array` and `map` wouldn't make sense, right?

Reply:
That would be the same for 113 or 1113. Nevertheless, they have any as data item as well.